posix/unix.c

```diff
@@ -373,6 +379,7 @@ int unix_accept4(unsigned socket, struct sockaddr *address, socklen_t *address_l
 		LIST_REMOVE(&s->connect, conn);
 		s->state &= ~US_ACCEPTING;
 
+		conn->state &= ~US_DISCONNECTED;
```

Comment:
maybe only clearing after sending SIGPIPE (+ add sigpipe sending to all socket operations?) to reset to "disconnected" state?

Reply:
I don't think SIGPIPE should change socket state, it's related only to send() call (based on POSIX spec).

Reply:
ok, it was just an idea to send `SIGPIPE` only once, if not needed - let's drop it.

Reply:
Turns out Linux forbids reconnecting stream socket, connect() call on peer disconnected socket return EISCONN.